synth/src/cli/stdf.rs

```diff
@@ -6,6 +6,7 @@ use serde_json::Value;
 
 use std::convert::TryFrom;
 use std::path::PathBuf;
+use crate::cli::json::samples_to_json;
 
 #[derive(Clone, Debug)]
 pub struct FileImportStrategy {
```

Comment:
Perhaps we should rename this `JsonImportStrategy` now that Json is no longer the default storage?